### PR TITLE
Fix CI format check

### PR DESCRIPTION
**Pull request by sophie-cluml**
Closes: #148

### CODE REVIEW

File: src/select/mini.rs [hunk 1]
```diff
@@ -381,9 +381,10 @@ where
                 format!("width: {w}px;")
             }
         });
-        let style_height = ctx.props()
-                .top_height
-                .map_or_else(String::new, |h| format!("height: {h}px;"));
+        let style_height = ctx
+            .props()
+            .top_height
+            .map_or_else(String::new, |h| format!("height: {h}px;"));
```
Comment (sehkone) [inline src/select/mini.rs:387]: Why has this been changed? I guess If we had been all in the same settings, it wouldn't have happened.

Comment (sophie-cluml) [reply src/select/mini.rs:387]: I ran below

```
cargo fmt -- --config group_imports=StdExternalCrate 
```

which caused the change in the format.

Comment (sehkone) [reply src/select/mini.rs:387]: It means the person who requested the previous PR, #145, didn't have the same settings in her/his VS Code. We need to track it down and ask her/him to get it fixed.

Comment (sophie-cluml) [reply src/select/mini.rs:387]: @jinjoolee07 Could you check above comment please, and setup your VS code settings accordingly?

Comment (jinjoolee07) [reply src/select/mini.rs:387]: @sophie-cluml checked all my settings, and everything is up to date!

**Pull request merged**